src/griddyn/CMakeLists.txt

```diff
@@ -557,13 +557,29 @@ add_library(
     ${sublibrary_files}
 )
 
+target_include_directories(
+    griddyn
+    PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+    # TODO is this right?
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
```

Comment:
I'm going to test squirrel against this next week, that should flush out any missing $<INSTALL_INTERFACE> parameters